fe/fe-core/src/main/java/com/starrocks/sql/ShortCircuitPlanner.java

```diff
@@ -158,7 +199,9 @@ protected static boolean isPointScan(Table table, List<String> keyColumns, List<
                     PartitionColumnFilter filter = filters.get(keyColumn);
                     if (filter.getInPredicateLiterals() != null) {
                         cardinality *= filter.getInPredicateLiterals().size();
-                        if (cardinality > MAX_RETURN_ROWS) {
+                        // TODO(limit operator place fe)
```

Comment:
what‘s the meaning of this comment?

Reply:
if limit compute node can be placed fe, we can treate fe as sink node. currently we only process return 1 record .  in other words, if having limit, cardinality must be 1